moveit_setup_assistant/src/tools/moveit_config_data.cpp

```diff
@@ -1142,78 +1146,81 @@ bool MoveItConfigData::inputROSControllersYAML(const std::string& file_path)
     YAML::Node doc = YAML::Load(input_stream);
     // Read the robot name
 
```

Comment:
this section of code is 3 for-loops deep... i think you should split the inner contents into a helper function to make it easier to read